lib/loadObj.js

```diff
@@ -495,6 +495,23 @@ function loadObj(objPath, options) {
         });
 }
 
+function getMtlPaths(mtllibLine) {
+    // Handle paths with spaces. E.g. mtllib my material file.mtl
+    var mtlPaths = [];
+    var splits = mtllibLine.split(' ');
+    var length = splits.length;
+    var startIndex = 0;
+    for (var i = 0; i < length; ++i) {
+        if (path.extname(splits[i]) === '') {
```

Comment:
Might be better to check if `!== ‘.mtl’`, something like `material.draft.mtl` could cause problems and might not be that unlikely if someone’s obj export software chooses names programmatically.